cypress/integration/viewer-grid.spec.ts

```diff
@@ -227,7 +234,39 @@ describe("Viewer Grid", () => {
         const calls = stub.getCalls();
 
         expect(calls.map(c => c.args[0]?.detail)).to.deep.equal(expected);
-      })
+      });
+  });
+
+  it('should say the right phrase uppon size change', () => {
```

Comment:
De correcte spelling is "upon". Daarnaast zie ik de verwachting explicieter: Welke phrase wordt er dan uitgesproken?